src/user/components/navbar/components/UserMenu/UserMenu.tsx

```diff
@@ -36,7 +36,6 @@ export function UserMenu() {
             history.push("/user/main/settings");
             setAnchorEl(null);
           }}
-          sx={{ pl: 20, pr: 20, pt: 3, pb: 3 }}
```

Comment:
I personally like the extra padding in the menu, the default one feels small to me, but I won't block because of this.

Reply:
I struggle to find a web page out there that has hugely padded menus like that, which is why I made it smaller...